.github/workflows/main.yml

```diff
@@ -157,4 +157,4 @@ jobs:
       - name: Publish logfire-api to PyPI
         uses: pypa/gh-action-pypi-publish@release/v1
         with:
-          packages-dir: logfire-api/
+          packages-dir: logfire-api/dist
```

Comment:
I think this was the only problem, but I can't predict if there will be another...